### PR TITLE
#09: add clean()

### PR DESCRIPTION
**Pull request by jansensan**
PR for [issue #9](https://github.com/kenzanmedia/pipeline-handyman/issues/9)

### CODE REVIEW

**Comment by jansensan** [comment]
Erh... conflicts but I can't see what they are? That's less than practical.

**Comment by obuckley-kenzan** [comment]
@jansensan 
Yeah, github doesn't show them in the PR, like bitbucket or stash would

**Comment by obuckley-kenzan** [comment]
@jansensan 
Please resolve the conflicts and include unit tests.  Ideally the setup would be to copy some fixtures into a direction, run clean, and confirm all the files are gone

File: package.json [hunk 1]
```diff
@@ -43,11 +43,12 @@
   "analyze": true,
   "license": "ISC",
   "dependencies": {
+    "del": "^2.0.2",
```
Comment (obuckley-kenzan) [inline package.json]: the version should be pinned here to `2.0.2`

File: README.md [hunk 2]
```diff
@@ -15,36 +16,57 @@ _repo_: `ssh://git@github.com:kenzanmedia/pipeline-handyman.git`
 
 _jenkins_: `https://kenzan.ci.cloudbees.com/job/CI-pipeline-handyman/`
 
+
 ## Install
+
 `npm install git+ssh://git@github.com/kenzanmedia/pipeline-handyman.git`
```
Comment (csharon) [inline README.md]: @obuckley-kenzan we should probably specify installing packages by name instead of using URL.

Comment (obuckley-kenzan) [inline README.md]: @csharon 
Agreed, but I think that is only applicable when these packages are in NPM, which is my plan for this week.  At which point I will update all the README's accordingly.

**Comment by jansensan** [comment]
@obuckley-kenzan @csharon I rebased the branch and fixed the build, do we still want to include this functionality?

**Comment by obuckley-kenzan** [comment]
@jansensan 
Had previously commented regarding the addition of unit testing.

**Comment by jansensan** [comment]
@obuckley-kenzan Ok, hadnt seen that unit testing part.

**Comment by jansensan** [comment]
@obuckley-kenzan @csharon Added unit test to the new `clean()` method.

**Comment by obuckley-kenzan** [comment]
test failures
```
Test Result (1 failure / +1)

should delete the path passed as a param.gulp-handyman clean should delete the path passed as a param
```

**Pull request closed without merging**